ntbs-integration-tests/ClinicalDetailsPageTests.cs

```diff
@@ -0,0 +1,282 @@
+            var document = await GetDocumentAsync(initialPage);
+
+            var formData = new Dictionary<string, string>
+            {
+                ["NotificationId"] = Utilities.DRAFT_ID.ToString(),
+                ["NotificationSiteMap[OTHER]"] = "true",
+                ["OtherSite.SiteDescription"] = "123",
+                ["ClinicalDetails.BCGVaccinationState"] = "Yes",
+                ["ClinicalDetails.BCGVaccinationYear"] = "1",
+                ["FormattedSymptomDate.Day"] = "10",
+                ["FormattedPresentationDate.Day"] = "1",
+                ["FormattedPresentationDate.Month"] = "1",
+                ["FormattedPresentationDate.Year"] = "2000",
+                ["ClinicalDetails.IsShortCourseTreatment"] = "true",
+                ["ClinicalDetails.IsMDRTreatment"] = "true",
+            };
+
+            // Act
+            var result = await SendFormWithData(document, formData);
+
+            // Assert
+            var resultDocument = await GetDocumentAsync(result);
+
+            result.EnsureSuccessStatusCode();
+            Assert.Equal(FullErrorMessage(ValidationMessages.StandardStringFormat), resultDocument.QuerySelector("span[id='other-site-error']").TextContent);
+            Assert.Equal(FullErrorMessage(ValidationMessages.ValidYear), resultDocument.QuerySelector("span[id='bcg-vaccination-error']").TextContent);
+            Assert.Equal(FullErrorMessage(ValidationMessages.ValidDate), resultDocument.QuerySelector("span[id='symptom-error']").TextContent);
+            Assert.Equal(FullErrorMessage(ValidationMessages.DateValidityRange(ValidDates.EarliestClinicalDate)),
+                resultDocument.QuerySelector("span[id='presentation-error']").TextContent);
+            Assert.Equal(FullErrorMessage(ValidationMessages.ValidTreatmentOptions), resultDocument.QuerySelector("span[id='short-course-error']").TextContent);
+            Assert.Equal(FullErrorMessage(ValidationMessages.ValidTreatmentOptions), resultDocument.QuerySelector("span[id='mdr-error']").TextContent);
+        }
+
+        [Fact]
+        public async Task PostNotified_ReturnsPageWithAllModelErrors_IfModelNotValid()
+        {
+            // Arrange
+            var initialPage = await client.GetAsync(GetPageRouteForId(Utilities.NOTIFIED_ID));
+            var document = await GetDocumentAsync(initialPage);
+
+            var formData = new Dictionary<string, string>
+            {
+                ["NotificationId"] = Utilities.NOTIFIED_ID.ToString(),
+                ["NotificationSiteMap[OTHER]"] = "true",
+                ["ClinicalDetails.BCGVaccinationState"] = "Yes",
+                ["ClinicalDetails.IsPostMortem"] = "true"
+            };
+
+            // Act
+            var result = await SendFormWithData(document, formData);
+
+            // Assert
+            var resultDocument = await GetDocumentAsync(result);
+
+            result.EnsureSuccessStatusCode();
+            // TODO: Check site error here after merging

```

Comment:
I think Farman fixed some bug on this page - something is definitely wrong with the site error validation on my branch so I'll wait until I merge to look into this